hybridcloud/util/metrics.py

```diff
@@ -0,0 +1,59 @@
+from typing import List
+from prometheus_client import Counter, Gauge
+from . import k8s
+import kopf
+
+PROMETHEUS_HANDLER_CALLS_TOTAL_COUNTER = Counter(
+    'handler_calls_total', 
+    'Amount of times a handler has been called', 
```

Comment:
Nitpick. Number of times sounds better than amount. Also for the other descriptions below.
```suggestion
    'Number of times a handler has been called', 
```